eeg_ds000117/sub-01/eeg/sub-01_task-facerecognition_coordsystem.json

```diff
@@ -1,8 +1,12 @@
 {
-  "EEGCoordinateSystem": "T1w",
-  "EEGCoordinateUnits": "mm",
-  "AnatomicalLandmarkCoordinates": {"LPA":[-0.072421 -7.0961e-09  7.7314e-09], "NAS":[-5.772e-09     0.10353  4.7199e-09], "RPA":[0.077791 -3.0159e-10  7.8728e-09]},
-  "AnatomicalLandmarkCoordinateSystem": "T1w",
```

Comment:
Previous CI fail was due to missing commas in array, which happened by reformatting from string. This is fixed now. Interestingly, a new error is revealed:

```
	1: Invalid JSON file. The file is not formatted according the schema. (code: 55 - JSON_SCHEMA_VALIDATION_ERROR)
		./sub-01/eeg/sub-01_task-facerecognition_coordsystem.json
			Evidence:  should have required property 'MEGCoordinateSystem'
		./sub-01/eeg/sub-01_task-facerecognition_coordsystem.json
			Evidence:  should have required property 'MEGCoordinateUnits'
		./sub-02/eeg/sub-02_task-facerecognition_coordsystem.json

```

This is potentially an issue with the eeg-validator not accepting EEGCoordinateSystem as well as MEGCoordinateSystem